app/src/androidTest/java/uk/gov/onelogin/login/ui/splash/SplashScreenTest.kt

```diff
@@ -43,25 +48,43 @@ class SplashScreenTest : TestCase() {
     @BindValue
     val mockSaveTokens: SaveTokens = mock()
 
+    @BindValue
+    val appInfoService: AppInfoService = mock()
+
     private lateinit var splashIcon: SemanticsMatcher
     private lateinit var unlockButton: SemanticsMatcher
     private lateinit var privacyNotice: SemanticsMatcher
+    private lateinit var loadingIndicator: SemanticsMatcher
+    private lateinit var loadingText: SemanticsMatcher
 
     @Before
     fun setUp() {
         hiltRule.inject()
 
+        wheneverBlocking { appInfoService.get() }.thenAnswer {
+            AppInfoServiceState.Successful(TestUtils.data)
+        }
+
         splashIcon = hasTestTag(resources.getString(R.string.splashIconTestTag))
         unlockButton = hasText(resources.getString(R.string.app_unlockButton))
         privacyNotice = hasTestTag(NOTICE_TAG)
+        loadingIndicator = hasContentDescription(
+            resources.getString(R.string.app_splashScreenLoadingContentDescription)
+        )
+        loadingText = hasText(resources.getString(R.string.app_splashScreenLoadingIndicatorText))
     }
 
     @Test
     fun verifySplashScreen() {
         // Given
+        wheneverBlocking { appInfoService.get() }.thenReturn(
```

Comment:
this already happens in the `@Before` ?

Reply:
Sorry, forgot to remove this after I added it to the `@Before`, done